src/Translator/src/TranslationsDumper.php

```diff
@@ -35,6 +35,7 @@ class TranslationsDumper
 
     public function __construct(
         private string $dumpDir,
+        private bool $dumpTypeScript,
         private MessageParametersExtractor $messageParametersExtractor,
```

Comment:
I now this class is marked @experimental but the component is out there for some times now ...  :/